docs/_running-tests/safari.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: page
+title: Safari
+---
+To run Safari on macOS, some manual setup is required:
+
+  * Disable "Block pop-up windows" in the security preferences.
+
+  * Ensure that "Allow Remote Automation" is enabled in the "Develop" menu.
```

Comment:
`safaridriver --enable` should work. (Throws admin prompt to authentication unless run as root, though.)

Reply:
Yes. Also, --enable is first supported in Safari 11.